# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jahshaka/qmake.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
qmake's (1.07a) include is buggy

--- jahshaka/jahshakaSettings.pro_orig	2005-08-06 14:14:15.000000000 +0200
+++ jahshaka/jahshakaSettings.pro	2005-11-08 21:43:49.000000000 +0100
@@ -9,7 +9,11 @@
 ###############################################################
 # This is the default config for all libraries and the app
 
-include( config.pro )
+JAHDEBUG=false
+JAHSTATIC=false
+JAHPREFIX=/usr
+JAHPLUGINBUILD=true
+BUILDTYPE=jahshaka
 
 ###System Options###
 JAHTHEMES       = true   #some linuxes have trouble with qt themes
